### PR TITLE
Part1

### PR DESCRIPTION
**Pull request by bobbybalow1**
Finished Assignment. Please Review.

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -56,86 +64,172 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
-    
-    
+    int secondNumber = 3;
+    int thirdNumber = 4;
+    char firstInitial = 66;
+    char lastInitial = 66;
+    char middleInitial = 66;
```
Comment (matkatmusic) [inline main.cpp]: `char` types are single alpha-numeric symbols wrapped in single quotes.   you are initializing a `char` with an integer.
- [x] revise.

File: main.cpp [hunk 2]
```diff
@@ -56,86 +64,172 @@ void variableDeclarations()
...
+    ignoreUnused(middleInitial);
+    ignoreUnused(thisIsTrue);
+    ignoreUnused(thisIsFalse);
+    ignoreUnused(isCorrect);
+    ignoreUnused(firstFloat);
+    ignoreUnused(secondFloat);
+    ignoreUnused(thirdFloat);
+    ignoreUnused(firstDouble);
+    ignoreUnused(secondDouble);
+    ignoreUnused(thirdDouble);
```
Comment (matkatmusic) [inline main.cpp]: - [x] call `ignoreUnused` once with all of your variables.

File: main.cpp [hunk 3]
```diff
@@ -56,86 +64,172 @@ void variableDeclarations()
...
+    { 
+    ignoreUnused(color, howMuchPaint); // Do I need to include this or just remove this because it returns void?
+    }
+
+//Function 3
+bool careForDog(int hasWater = 1, int hasFood = 0, int hasWalk = 0)
+    { 
+    ignoreUnused(hasWater, hasFood, hasWalk);
+    return {}; //function returns bool.
+    }
```
Comment (matkatmusic) [inline main.cpp]: Fix your `{ }` placement.  see the style guide in Readme.MD if you are unsure where they should go.
- [x] revise

File: main.cpp [hunk 4]
```diff
@@ -56,86 +64,172 @@ void variableDeclarations()
...
+    auto carRented = rentACar(6, 2); 
     
     //1)
+
+    auto workedForYears = yearsWorking(); //already initiailized
     
     //2)
     
+    //Returned Void
+
```
Comment (matkatmusic) [inline main.cpp]: You still need to call them even if they don't return a value.
- [x] add the function you didn't call.

File: main.cpp [hunk 5]
```diff
@@ -56,86 +64,172 @@ void variableDeclarations()
...
+    auto dogCaredFor = careForDog(); //already initialized
+    
     //4)
     
+    auto finishedDinner = eatDinner(); // already initialized
+    
     //5)
     
+    //Returned Void
+
```
Comment (matkatmusic) [inline main.cpp]: You still need to call them even if they don't return a value.
- [x] add the function you didn't call.

File: main.cpp [hunk 6]
```diff
@@ -56,86 +64,172 @@ void variableDeclarations()
...
+    //IGNORE FUNCTIONS
+    ignoreUnused(carRented);
+    ignoreUnused(workedForYears);
+    ignoreUnused(dogCaredFor);
+    ignoreUnused(finishedDinner);
+    ignoreUnused(eligibleForHiring);
+    ignoreUnused(madeInvestment);
+    ignoreUnused(songRecorded);
+    ignoreUnused(userSound);
+    ignoreUnused(completedHomework);
```
Comment (matkatmusic) [inline main.cpp]: - [x] call `ignoreUnused()` once with all of your variables.

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (bobbybalow1) [reply main.cpp]: When would you use the char number? For example 66 is "B"?

File: main.cpp (hunk 3 above)
Comment (bobbybalow1) [reply main.cpp]: Thank you for being thorough! Will do.

File: main.cpp [hunk 7]
```diff
@@ -79,51 +79,55 @@ void variableDeclarations()
...
-    ignoreUnused(thisIsTrue);
-    ignoreUnused(thisIsFalse);
-    ignoreUnused(isCorrect);
-    ignoreUnused(firstFloat);
-    ignoreUnused(secondFloat);
-    ignoreUnused(thirdFloat);
-    ignoreUnused(firstDouble);
-    ignoreUnused(secondDouble);
-    ignoreUnused(thirdDouble);
+    ignoreUnused
```
Comment (matkatmusic) [inline main.cpp]: No, this notation is not recommended.   you would use this notation with curly braces `{ }` if you were initializing an object or some container that took a `std::initializer_list` as a function parameter.
- [x] revise.

File: main.cpp [hunk 8]
```diff
@@ -220,15 +224,18 @@ int main()
...
+        carRented,
+        workedForYears,
+        dogCaredFor,
+        finishedDinner,
+        eligibleForHiring,
+        madeInvestment,
+        songRecorded,
+        userSound,
+        completedHomework
+    );
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise.  don't use this style for passing multiple arguments to a function.

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (bobbybalow1) [reply main.cpp]: image

This must be for something completely different 😀

Comment (bobbybalow1) [reply main.cpp]: But this is what my "dec" number was referring to.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**